src/content/content.service.ts

```diff
@@ -57,6 +61,35 @@ export class ContentService {
     return transformContent(ContentType.glossaryItem, entries)
   }
 
+  async getAiContentPrompts() {
+    const prompts = await this.prisma.content.findMany({
+      where: {
+        OR: [
+          {
+            type: ContentType.onboardingPrompts,
+          },
+          {
+            type: ContentType.candidateContentPrompts,
+          },
+        ],
+      },
+    })
+
+    // should be one content record for each "type" of prompt
+    if (prompts.length !== 2) {
+      throw new NotFoundException('Prompt content not found')
```

Comment:
This should be a `BadGatewayException` since this has to do w/ our integration with Contentful, not something the requesting service is asking for.

Reply:
I would also do a check for `.data` on both prompts here, and that way the try-catch below can be removed.

Reply:
It's data effectively cached from a third-party service. If we weren't storing that data in our DB, and just calling Contentful directly, we'd be returning `BadGatewayException` here, not a 404. 

This method is also only called by the `CampaignsAiService` service, so it's not being called by the client.  A 404, bubbling up through the `CampaignsAiService` would give the impression that the `aiContent` for the given Campaign isn't there, when that's not what's happening there, the _Content_ for to generate new `aiContent` are missing/corrupted.  That's a 502 for that integration.